Doc/c-api/init.rst

```diff
@@ -564,7 +564,7 @@ Additionally, when extending or embedding Python, calling :c:func:`fork`
 directly rather than through :func:`os.fork` (and returning to or calling
 into Python) may result in a deadlock by one of Python's internal locks
 being held by a thread that is defunct after the fork.
-:c:func:`PyOS_AfterFork` tries to reset the necessary locks, but is not
+:c:func:`PyOS_AfterFork_Child` tries to reset the necessary locks, but is not
```

Comment:
"there is no way to inform Python of additional (non-Python) locks that need to be acquired before or reset after a fork" and mentioning `pthread_atfork`.